### DISCUSSION TITLE
Slim side panel mode

### DISCUSSION THREAD

**Discussion by SteveEdson** [spatie/ray#128] [Ideas]
Just bought Ray, it's awesome, thank you.

I'd really like it to be tweaked to it fits better in a slim side panel layout. Currently the timestamp still takes up 50% of the row, leaving little space to see the important data. It would be better if these stacked to display more info.

<img width="512" alt="Screenshot 2021-01-12 at 20 35 01" src="https://user-images.githubusercontent.com/1201960/104369631-a7092280-5515-11eb-8735-805dfb64ebf2.png">

**Comment by AdrianMrn** [comment]
Thanks for the feedback. We'll discuss this further internally!

**Comment by slaFFik** [comment]
Yeah, very valid point.

**Comment by AdrianMrn** [comment]
@willemvb did some fantastic work on this last week. I'll add some finishing touches this week and release it!

**Reply by SteveEdson** [reply]
Excellent thanks, looking forward to it!
